src/pixelmapFeature.js

```diff
@@ -339,13 +339,14 @@ var pixelmapFeature = function (arg) {
       m_quadFeature = m_this.layer().createFeature('quad', {
         selectionAPI: false,
         gcs: m_this.gcs(),
-        visible: m_this.visible()
+        visible: m_this.visible(undefined, true)
```

Comment:
it would have been nice to just call it like m_this.visible(true). Should we switch the args placeholder?